bin/dopublish.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+kob_rel_version=$1
+kob_archive_download_repo="KOBman_target_repo"
+branch="REL-$kob_rel_version"
+kob_namespace= ${KOB_NAMESPACE:-hyperledgerkochi}
+#Checkout latest tag
+# The branch is used to make the tar files.
+git branch -D $kob_rel_version-branch
```

Comment:
Branch creation should be done in release. 

You need to only check if the branch is available , if mention nothing is available to publish.